apps/server/src/utils/__tests__/parserFunctions.test.ts

```diff
@@ -11,19 +11,20 @@ describe('sanitiseOscSubscriptions()', () => {
 
   it('returns an array of valid entries', () => {
     const oscSubscriptions: OscSubscription[] = [
-      { id: '1', cycle: 'onLoad', message: 'test', enabled: true },
-      { id: '2', cycle: 'onStart', message: 'test', enabled: false },
-      { id: '3', cycle: 'onPause', message: 'test', enabled: true },
-      { id: '4', cycle: 'onStop', message: 'test', enabled: false },
-      { id: '5', cycle: 'onUpdate', message: 'test', enabled: true },
-      { id: '6', cycle: 'onFinish', message: 'test', enabled: false },
+      { id: '1', cycle: 'onLoad', path: '/test', message: 'test', enabled: true },
```

Comment:
Can you explain what the advantage is in separing the message from the path? 
according to specification, the osc payload contains a single element `address`

Reply:
hmmm it is normally an adders and some values
https://github.com/colinbdclark/osc.js/?tab=readme-ov-file#sending-osc-messages

a simple thing like `/go` can often be just the address but if you wanted to do something like setting a fader
you would often do `/fader/6 100` thereby setting fade 6 to 100

Reply:
It sounds that what we need is an address and a payload, not an address and a message 

Combination of address and payload is already what we do, I guess the payload is not directly exposed to the user since we only allow it to be an ontime value which we have control over

Reply:
yeah we can rename it to address and payload

before there was no payload, the ontime value was sent as part of the address

I'm thinking if we should allow the user to choose the payload type or maybe string is enough for now
as the template parser always returns a string anyways

Reply:
yes it has always been empty